apps/dashboard/src/app/api/engine-status/error-rate/route.ts

```diff
@@ -0,0 +1,73 @@
+import { client, v2 } from "@datadog/datadog-api-client";
+import { NextResponse } from "next/server";
+
+export const POST = async () => {
```

Comment:
for now this makes sense, when we move the `engine` part of dashboard to app router we will be able to query this directly in server components (but we can't do that in pages so this is a good workaround for now)

Reply:
technically this could be a GET request?

Reply:
do we need some kind of auth check to access this